pyproject.toml

```diff
@@ -30,6 +30,7 @@ documentation = "https://eliotwrobson.github.io/CyNetDiff/"
 
 [tool.poetry.dependencies]
 python = ">=3.9"
+jupyter = "^1.0.0"
```

Comment:
Two things
1. It looks like this is the old Jupyter release, the newly maintained one is here: https://pypi.org/project/notebook/
2. Could you put this under the benchmark dependencies and update the lockfile? This general dependency section is for anyone installing this and we want to keep the total dependencies as minimal as possible.

Reply:
I wasn't sure how to update the lockfile

Reply:
Run `poetry lock --no-update`